### DISCUSSION TITLE
Customize PHP-FPM port

### DISCUSSION THREAD

**Discussion by alucryd** [arabcoders/watchstate#452] [General] [answered]
Hi there, I'm in a joint emby/jellyfin environment and would like to synchronize more than one users. As it's not recommended to filter by user id for these 2 I need to spin several containers (via podman). As I want to keep things tidy, I'd like to have all of them in the same pod, however there are conflicts because PHP-FPM runs on port 9000 on all of them. Would it be possible to add an env variable that would allow this port to be customized?

**Comment by arabcoders** [comment]
Hi, you probably shouldn't expose the 9000 port unless you are disabling the internal caddyserver and doing external fcgi proxy. 

However i will see if your case is possible.

**Reply by alucryd** [reply]
Thanks for looking into it. It's not exposed, however containers in the same pod are sharing the same network, so they can't use identical ports.

**Reply by arabcoders** [reply, marked as answer]
Alright, Once #453 is merged into master you will be able to change the FPM port by using `FPM_PORT` env, as it's system level, it has to be from the `compose.yaml` file. it cannot be used via `/config/config/.env` like the other system level ENV. [Read more here](https://github.com/arabcoders/watchstate/blob/master/FAQ.md#container-specific-environment-variables)

**Reply by alucryd** [reply]
Awesome, thank you!
